test/e2e/httpserver_test.go

```diff
@@ -1,19 +1,82 @@
+// +build all virtual
```

Comment:
can you think of a better name of this file? Its not a great one by me. I want to keep it for just testing additional network resource injection only. Its too generic with httpserver_test.go. My bad.